### ISSUE TITLE
Program files and libraries should not be stored in /usr/share

### ISSUE THREAD

**Issue by MingcongBai** [minbrowser/min#450]
In the official .deb packages, program files and libraries (/usr/share/min) are stored in /usr/share, however, as per [FHS](http://www.pathname.com/fhs/pub/fhs-2.3.pdf) chapter 4.11...

```
The /usr/share hierarchy is for all read-only architecture independent data files.
```

I would suggest moving `/usr/share/min` to `/usr/lib/min`.

**Comment by PalmerAL** [comment]
It looks like we need to upgrade ```grunt-electron-installer-debian``` to 0.5.0 to include https://github.com/unindented/electron-installer-debian/pull/47.

**Comment by PalmerAL** [comment]
This is fixed in 834aea269be9bbb0a4765a7abd9dc4e2a1b2586a.

**Comment by PalmerAL** [comment]
The 1.7.0 version that includes this fix has been released, you can download it from here: https://github.com/minbrowser/min/releases/tag/v1.7.0.
